setup.py

```diff
@@ -132,6 +132,7 @@
     "gguf>=0.10.0",
     "torchao>=0.7.0",
     "bitsandbytes>=0.43.3",
+    "nvidia_modelopt[torch, hf]>=0.27.0",
```

Comment:
```suggestion
    "nvidia_modelopt[torch]>=0.27.0",
```
I think only torch optional deps are needed here. Diffusers is anyways installed here and I think diffusers depending on transformers, accelerate, datasets, etc hf dependencies doesn't make sense

Reply:
> Diffusers is anyways installed here and I think diffusers depending indirectly on transformers, accelerate, datasets, etc hf dependencies doesn't make sense.

`transformers`, `datasets` are not required packages `diffusers` just as an FYI. We prefer keeping our dependencies lean.

Reply:
keeping this as it is for now, lmkyt ?